### PR TITLE
elf-loader: Switch to any start hart instead of 1

### PR DESCRIPTION
**Pull request by yyshen**
This change switches to any start hart specified by CONFIG_FIRST_HART_ID instead of fixed id 1.

Signed-off-by: Yanyan Shen <yshen@hybridkernel.com>

### CODE REVIEW

File: elfloader-tool/src/arch-riscv/crt0.S [hunk 1]
```diff
@@ -62,7 +62,7 @@ _start:
 hsm_switch_hart:
   li a7, SBI_HSM_BASE
   li a6, SBI_HSM_BASE_HART_START
-  li a0, 1       // hart id to start
+  li a0, CONFIG_FIRST_HART_ID       // hart id to start
```
Comment (axel-h) [inline elfloader-tool/src/arch-riscv/crt0.S:65]: Please fix the comment `/* start hart 1*/` then also.

Comment (yyshen) [reply elfloader-tool/src/arch-riscv/crt0.S:65]: @axel-h  fixed

**Review by axel-h — APPROVED** [review]
looks ok to me.

**Review by lsf37 — APPROVED** [review]

**Comment by yyshen** [comment]
@lsf37 ok to merge?

**Comment by yyshen** [comment]
> @lsf37 ok to merge?

Thanks. next time I will click the merge button :-)

**Pull request merged**